### PR TITLE
Refactor/#114 use snack bar with redux

### PR DESCRIPTION
**Pull request by revodark**

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
 import React from 'react';
-import { createStore } from 'redux';
+import { createStore, combineReducers } from 'redux';
 import { Provider } from 'react-redux';
 
 import { StylesProvider } from '@material-ui/core/styles';
```
Comment (Rolly007) [inline frontend/src/App.js:5]: Please, order your imports

File: frontend/src/components/SnackBar/SnackBar.jsx [hunk 2]
```diff
@@ -1,25 +1,30 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
 
-import Snackbar from '@material-ui/core/Snackbar';
-import SnackbarContent from '@material-ui/core/SnackbarContent';
+import { Snackbar, SnackbarContent, IconButton } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
```
Comment (Rolly007) [inline frontend/src/components/SnackBar/SnackBar.jsx:5]: Please, import material ui Icons in the "utils/icons/index.js" file, and use ir from there

File: frontend/src/routes/AppRoutes.test.js [hunk 3]
```diff
@@ -6,8 +6,11 @@ import * as redux from 'react-redux';
 
 import AppRoutes from './AppRoutes';
 
-const spy = jest.spyOn(redux, 'useSelector');
-spy.mockReturnValue({ username: 'pedro Rodriguez', role: 'standard' });
+const spySelector = jest.spyOn(redux, 'useSelector');
```
Comment (Rolly007) [inline frontend/src/routes/AppRoutes.test.js:9]: I am not sure but maybe you can mock "useSelector" without using jest-Spy only using the jest.mock('react-redux') and  (useSelector.mockReturnValue({ username: 'pedro Rodriguez', role: 'standard' }))

File: frontend/src/views/Login/Login.jsx [hunk 4]
```diff
@@ -49,52 +51,45 @@ const formFields = [
 
 function Login() {
   const history = useHistory();
-  const [open, setOpen] = useState(false);
-  const [{ error }, executeLogin] = useAxiosRequest(config);
+  const setErrorSnackBar = useSnackBar(ERROR);
```
Comment (Rolly007) [inline frontend/src/views/Login/Login.jsx:54]: are you sure if the snackBar will be render from login form?, beacuse you are calling to useSnackbar in the AppRoutes component, and login is in the "routes/index.js" file, Please could you test it

File: frontend/src/views/RegisterUser/RegisterUser.jsx [hunk 5]
```diff
@@ -42,13 +47,7 @@ function RegisterUser() {
...
         history.push('/movies');
       })
-      .catch(() => {
-        setSnackBar({
-          type: 'error',
-          message: 'Something happend, please try again'
-        });
-        setOpen(true);
-      });
+      .catch(() => setErrorSnackBar('Something happend, please try again'));
```
Comment (Rolly007) [inline frontend/src/views/RegisterUser/RegisterUser.jsx:50]: TYPO =>''happened"

File: frontend/src/routes/AppRoutes.test.js (hunk 3 above)
Comment (revodark) [reply frontend/src/routes/AppRoutes.test.js:9]: Yes, is better, so we can have consistency with the other tests

File: frontend/src/components/SnackBar/SnackBar.jsx (hunk 2 above)
Comment (revodark) [reply frontend/src/components/SnackBar/SnackBar.jsx:5]: thanks!

File: frontend/src/views/Login/Login.jsx (hunk 4 above)
Comment (revodark) [reply frontend/src/views/Login/Login.jsx:54]: totally right! thanks

**Pull request merged**